product_docs/docs/biganimal/release/getting_started/identity_provider/azure_ad.mdx

```diff
@@ -0,0 +1,59 @@
+1. From the left navigation pane, select **Enterprise Applications**. On the Enterprise Applications page: 
+   1. Select **New application > Create your own application**.
+
+   1. Enter a name for your application.
+   1. Select the third choice for what you intend to do with your application: **Integrate any other application you don’t find in the gallery (Non-gallery)**.
+1. After the application is created, from the left navigation menu, select **Single sign-on**. On the Single sign-on page: 
+   1. Select **SAML** as your single sign-on method.
+   1. Update the Basic SAML Configuration by copying and pasting the following information from the Set Up Identity Provider page in BigAnimal to the SAML Configuration menu in Azure AD:
+
+      | Copy from BigAnimal            | Paste in SAML Configuration |
+      | ------------------------------ | --------------------------- |
+      | Audience URI                   | Identifier (Entity ID)      |
+      | Assertion Consumer Service URL | Reply URL                   |
+   
+   1. Enter the configuration for Attributes & Claims. We recommend the following:
+
+      | Claim name                       | Value                                               | Note             |
+      |----------------------------------|-----------------------------------------------------|------------------|
+      | Unique User Identifier (Name ID) | user.userprincipalname [nameid-format:emailAddress] | Required claim   |
+      | &lt;assertion_path>/emailaddress | user.mail                                           | Additional claim |
+      | &lt;assertion_path>/givenname    | user.givenname                                      | Additional claim |
+      | &lt;assertion_path>/name         | user.displayname                                    | Additional claim |
+      | &lt;assertion_path>/surname      | user.surname                                        | Additional claim |
+   
+      Where &lt;assertion_path> is http://schemas.xmlsoap.org/ws/2005/05/identity/claims.
+
+   1. Under **SAML Signing Certificate**, select **Download** for the Base64-encoded certificate.
+   1. Copy the **Login URL** value.
+1. From the left navigation pane, Select **Properties**. On the Properties page, if you don't want all of the users in Azure AD to use BigAnimal, answer `No` for the **Assignment required?** question. Otherwise, answer `Yes` and assign the specified users to this Enterprise Application explicitly on the Users and Groups page.
+1. On the **Setup Config** tab on the Set Up Identity Provider page in BigAnimal: 
+   1. Paste the Login URL value you copied from the Single sign-on page in Azure AD as the **Single Sign-On URL**.
+   1. For **Identity Provider Signature Certificate**, upload the Base64-encoded certificate downloaded from Azure.
+   1. Select the appropriate method for **Request Binding**. Azure AD supports HTTP POST and HTTP Redirect. 
```

Comment:
Azure AD supports HTTP-Post and Hybrid 

(HTTP-Redirect choice is not supported by Azure AD per our test).

Reply:
yes it is.

<img width="752" alt="image" src="https://user-images.githubusercontent.com/79249661/167148064-3afcc837-a55c-4aca-bb47-2c45352197f1.png">

Reply:
fixed